frontend/packages/integration-tests-cypress/tests/crud/namespace-crud.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { checkErrors, testName } from '../../support';
+import { listPage } from '../../views/list-page';
+import { modal } from '../../views/modal';
+
+describe('Namespace', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  const newName = `${testName}-ns`;
+
+  it('Create, Retrieve (list), Update (labels), Delete', () => {
```

Comment:
We should try to be consistent with our test description

```suggestion
  it('creates, edits labels, and deletes', () => {
```

Reply:
Was trying to illustrate 'CRUD' usage.

Reply:
fixed